downstream/modules/automation-hub/proc-configure-ansible-galaxy-cli-verify.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc-configure-ansible-galaxy-cli-verify_{context}"]
+
+= Configuring Ansible-Galaxy CLI to verify collections
+
+You can configure Ansible-Galaxy CLI to verify collections. This assures that collections you download are approved by your organization and have not been changed after it was uploaded to  {HubName}.
+
+.Prerequisites
+
+* Signed and certified collections have been published to {HubName}.
+* Public key for verification has been added to the local system keyring.
+
+.Procedure
+
+If a collection has been signed by the Galaxy server, the server provides ASCII armored, detached signatures to verify the authenticity of the MANIFEST.json before using it to verify the collection’s contents. You must opt into signature verification by link:https://docs.ansible.com/ansible/devel/reference_appendices/config.html#galaxy-gpg-keyring[configuring a keyring] for `ansible-galaxy` or providing the path with the `--keyring` option.
+
+. To import a public key into a keyring for use with `ansible-galaxy`, run the following command.
++
+----
+gpg --import --no-default-keyring --keyring ~/.ansible/pubring.kbx my-public-key.asc
+----
++
+[NOTE]
+====
+In addition to any signatures provided by the Galaxy server, signature sources can also be provided in the requirements file and on the command line. Signature sources should be URIs.
+====
++
+. Use the --signature option to verify collection name(s) provided on the CLI with an additional signature.
```

Comment:
I am unsure if this explains the usage properly here, this almost feels like users who don't understand gpg won't be able to understand how this works. but I maybe wrong, maybe @sivel can help

Reply:
@anshulbehl - I do agree with you here. I am just limited by not having a sandbox or something that displays the process more clearly to me. I thought I could rely on the Signing content collectins exercise you designed in instruqt, but that is set up a bit differently. 
@sivel - Can you provide more insight here? Would greatly appreciate your expertise.

Reply:
This documentation does feel a little hollow, especially since I don't see any directly linked documentation on creating additional signatures.  To be honest, the `--signature` CLI flag, as well as the ability to place signatures into `requirements.yml` was more of a community feature than one intended for customer use, since the community will not get access to a galaxy server that has signing capabilities.

It's probably still useful to a very small percentage of customers, but without clarification about why they may want to do this instead of just utilizing on-prem hub for signatures, this basically is a footnote that this is possible functionality.

Reply:
Will bring this up again with Dylan Silva.